### DISCUSSION TITLE
Halos and clusters

### DISCUSSION THREAD

**Discussion by Lucia-Fonseca** [skypyproject/skypy#534] [Ideas]
Halos and clusters: next step: feature of baryonic component in galaxy cluster, e.g. X-ray luminosity, X-ray temperature, gas mass.

@sutieng please feel free to add any more information to this topic. Thanks!

**Comment by Lucia-Fonseca** [comment]
@sutieng would you have any suggestion?

**Reply by sutieng** [reply]
The current halo module has already implemented some characteristic DM properties for cluster-scale halos by assuming NFW model. For the baryonic component, to obtain the full X-ray brightness distribution, it would require some hydrodynamical simulations. Alternatively, we could use scaling relations to connect DM with baryons, and paint each sampled NFW halo with X-ray properties.
